src/bahmni-helm/templates/apps/openmrs/openmrs-deployment.yml

```diff
@@ -86,7 +86,16 @@ spec:
             - name: {{ .Release.Name }}-data
               subPath: "openmrs/configuration_checksums"
               mountPath: "/usr/local/tomcat/.OpenMRS/configuration_checksums/"
-            - name: haiti-data
+            - name: {{ .Release.Name }}-data
+              subPath: "openmrs/lucene"
+              mountPath: "/usr/local/tomcat/.OpenMRS/lucene/"
+            - name: {{ .Release.Name }}-data
+              subPath: "openmrs/activemq-data"
+              mountPath: "/usr/local/tomcat/.OpenMRS/activemq-data/"
+            - name: {{ .Release.Name }}-data
+              subPath: "openmrs/owa"
+              mountPath: "/usr/local/tomcat/.OpenMRS/owa/"
+            - name: {{ .Release.Name }}-data
```

Comment:
I don't think this is "data" per say. This is rather meant to deploy artifacts so fits into the distro volume instead. But right now our distros don't provide any OWAs so the path is absent. But we should keep the support for it.